crates/nu-command/src/commands/conversions/into/filepath.rs

```diff
@@ -0,0 +1,126 @@
+use std::path::PathBuf;
+
+use crate::prelude::*;
+use nu_engine::WholeStreamCommand;
+use nu_errors::ShellError;
+use nu_protocol::{ColumnPath, Primitive, Signature, SyntaxShape, UntaggedValue, Value};
+
+pub struct SubCommand;
+
+impl WholeStreamCommand for SubCommand {
+    fn name(&self) -> &str {
+        "into path"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("into path").rest(
+            SyntaxShape::ColumnPath,
+            "column paths to convert to filepath (for table input)",
```

Comment:
Usage and the signature here would be more consistent using `path` but the signature uses both `column path` and `path` then, so not sure what less confusing to the user.

Reply:
Column path means the table column name, e.g., in `... | into path name`, "name" is the column path. I think here "filepath" makes it stand out more against "column path" than it would with just "path", I'd keep it.

Not sure if I understood your question.

Reply:
Thanks, that's what I meant. :smile: